docs/docs/tutorials/login-with-keycloak.md

```diff
@@ -0,0 +1,121 @@
+---
+id: login-keycloak
+title: Authenticate and launch JupyterLab
+description: |
+  A step-by-step guide to logging into your Nebari instance and launching a JupyterLab pod.
+---
+
+# Authenticate and launch JupyterLab
+
+This guide provides a basic overview of how to log in to your Nebari instance (through Keycloak) and launch JupyterLab on a new server (ie., machine).
+
+## 1. Connect to your Nebari instance
+
+Navigate to the instance homepage.
+
+Once on the site, you will be prompted to log in, similar to
+the login page shown in the image below.
+
+![Nebari login screen](/img/how-tos/nebari_login_screen.png)
+
+Nebari uses [Keycloak](https://www.keycloak.org/), an open-source identity and access management tool. Keycloak is a centralized location for administrators to add new users, create user groups and update roles. For more information, see the [How to Configure Keycloak][nebari-keycloak] docs.
+
+:::warning
+Nebari **does not** create a "regular" user for you during deployment, it only creates the `root` Keycloak admin user. The following steps assume that your Nebari administrator has already created a Keycloak user in the Keycloak admin console for you. If not, follow the steps in [Adding new users to Keycloak][nebari-keycloak-add-user] to create a Keycloak user.
+:::
+
+## 2. Authenticate with your provider
+
+At this stage, it is a little challenging to provide thorough instructions because the particulars will depend on your [identity provider](https://www.keycloak.org/docs/latest/server_admin/#_identity_broker) / authentication provider ([LDAP](https://pt.wikipedia.org/wiki/LDAP), [OAuth 2.0](https://oauth.net/2/), passwordless authentication, password-based authentication and many others).
+
+For more information on how to configure Keycloak, and add new users, review the [How to configure Keycloak][nebari-keycloak] sections of the docs.
+
+In this example, Auth0 is the identity provider.
+To continue the login process,
+select the `Auth0` button and login.
+
+<p align="center">
+<img src="/img/tutorials/keycloak_nebari_login.png" alt="Nebari Keycloak auth screen - shows a form to provide username or password or to authenticate through Auth0" width="400"/>
+</p>
+
+## 3. Navigate the landing page
+
+:::note
+You may need to "Authorize access" to reach the home page:
+
+<p align="center">
+<img src="/img/tutorials/jhub-apps-authorize.png" alt="Authorize access page with an 'Authorize' button requesting access for JHub Apps, which is a Jupyter Service, to identify the user." width="80%"/>
+</p>
+:::
+
+Once authenticated, the **Nebari home page** opens.
+The home page, also referred to as "Landing page" or "JupyterHub launcher",
+lists Nebari's core **services** and allows you to create and share web apps.
+
+<!-- TODO: Link to app creation & sharing tutorial when available -->
+
+Click on the **"JupyterLab"** button under Services to start a server and launch JupyterLab:
```

Comment:
JupyterLab is now only available under _My Apps_. This section & the image need updating.